saleor/graphql/webhook/subscription_types.py

```diff
@@ -2396,6 +2396,46 @@ class Meta:
         description = "Event sent when voucher is deleted." + ADDED_IN_34
 
 
+class VoucherCodeBase(AbstractType):
+    voucher_code = graphene.Field(
+        "saleor.graphql.discount.types.VoucherCode",
+        description="The voucher code the event relates to.",
+    )
+    voucher = graphene.Field(
+        "saleor.graphql.discount.types.Voucher",
+        channel=graphene.String(
+            description="Slug of a channel for which the data should be returned."
+        ),
+        description="The voucher the code relates to.",
+    )
+
+    @staticmethod
+    def resolve_voucher_code(root, _info: ResolveInfo):
+        _, voucher_code = root
+        return voucher_code
+
+    @staticmethod
+    def resolve_voucher(root, _info: ResolveInfo):
+        _, voucher_code = root
+        return ChannelContext(node=voucher_code.voucher, channel_slug=None)
+
+
+class VoucherCodeCreated(SubscriptionObjectType, VoucherCodeBase):
```

Comment:
@korycins so:
 -  `VoucherCodesCreated` and `VoucherCodesDeleted` were added to send one bulk event instead of one per created/deleted code
 - `VoucherUpdated` will be sent only when the Voucher instance is updated, so adding new codes will not trigger this event
 - `voucherDelete` mutation will send only `voucherDeleted` event